joblib/test/test_memory.py

```diff
@@ -20,12 +20,22 @@
 from joblib.memory import _load_output, _get_func_fullname
 from joblib.memory import JobLibCollisionWarning
 from joblib.test.common import with_numpy, np
-from joblib.testing import raises, warns
+from joblib.testing import fixture, parametrize, raises, warns
 from joblib._compat import PY3_OR_LATER
 
 
 ###############################################################################
-# Module-level variables for the tests
+# Module-level functions and fixture, for tests
+@fixture(scope='function')
+def memory(tmpdir):
+    return Memory(cachedir=tmpdir.strpath, mmap_mode=None, verbose=0)
```

Comment:
Not convinced by the `memory` and `no_memoize_memory` fixture, they only replace a one-line statement and add a layer of indirection. I feel it is better in this case to be explicit and go for simplicity.